tabular/src/autogluon/tabular/task/tabular_prediction/__init__.py

```diff
@@ -1,3 +1,4 @@
 from .tabular_prediction import *
 from .dataset import *
-from .predictor import * 
+from .predictor_legacy import *
```

Comment:
Is this going to be removed/renamed/refactored? Any good reason we want to keep legacy code active?

Reply:
The plan is to remove it in Part 5, along with TabularPrediction